### DISCUSSION TITLE
Can't find my commit count on `Insights/contributors` page.

### DISCUSSION THREAD

**Discussion by Hisoka-X** [community/community#55427] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I can find my commit info in my profile page and Insights/Pulse page. But can't find it in Insights/contributors page. There two project for me, The Apache SeaTunnel work fine, but Apache Spark work not fine for me. My git config are same for there two repo.

**Comment by chamale-rac** [comment]
Hello Hisoka-X,

It sounds like you're having trouble finding your commit info in the contributors page for the Apache Spark project. There could be several reasons why you're not appearing in the contributors graph, as you mentioned. If your commits haven't been merged into the default branch, then they won't show up in the contributors graph. If you're not one of the top 100 contributors, then your contributions may not appear on the graph either.

One thing you can try is to view contributors during a specific time period. To do this, click on the "Contributors" tab in the Insights section of the repository, then click and drag until the time period you want to view is selected. The contributors graph sums weekly commit numbers onto each Sunday, so your time period must include a Sunday.

Another option is to use a third-party tool like the [GitHub Action "Contribute List"](https://github.com/marketplace/actions/contribute-list). This tool can help you list all the contributors in the readme of your project. However, keep in mind that this is not an official GitHub feature and may not be suitable for all projects.

I hope this helps! Let me know if you have any further questions or concerns.

Best regards,
Samuel

References:
[https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository/viewing-a-projects-contributors](https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository/viewing-a-projects-contributors)

**Reply by Hisoka-X** [reply]
@chamale-rac 

> To do this, click on the "Contributors" tab in the Insights section of the repository, then click and drag until the time period you want to view is selected.

I already select last year, and some PR already merge into master (default branch). There are no my commit info

there are commit info by select time range

https://github.com/apache/spark/graphs/contributors?from=2021-07-22&to=2023-05-15&type=c

there are my commit

https://github.com/apache/spark/commits?author=Hisoka-X
